net/pfSense-pkg-cellular/files/usr/local/www/widgets/widgets/cellular.widget.php

```diff
@@ -0,0 +1,237 @@
+<?php
+@require_once("guiconfig.inc");
+@require_once("pfsense-utils.inc");
+@require_once("functions.inc");
+@require_once("/usr/local/www/widgets/include/interfaces.inc");
+
+global $config;
+$python_bin = '/usr/local/bin/python2.7';
+$interface_bin = '/usr/local/sbin/cellular';
+$widget_url = '/widgets/widgets/cellular.widget.php';
+
```

Comment:
We are trying to convert pfSense to POST operation wherever possible. Perhaps it would be best to start of that way here?